
<html>
<head>
  <title> Metadata spreadsheet </title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
<link rel="stylesheet" type="text/css" href="df_style.css"/>
</head>
<body>
<h1> List of Metadata fields </h1>

<p>
Version: 21.06-1-rc1 <br />
Date: 2021-06-07 <br />
</p>

<p>
<h2> Download spreadsheets</h2>
<ul>
<li> <a href="sheets/sequencing_spreadsheet_template.by_provider.xlsx"> ["classic"] Sorted by Provider (ILSe, etc) </a> </li> 
<li> <a href="sheets/sequencing_spreadsheet_template.by_category.xlsx"> [new] Sorted by category (Sample, Experiment, Sequencing Info, LOTs) </a> </li> 
</ul>

</p>
<p>
In case of questions please open a issue at <a href="https://github.com/odomlab2/metadata-submission/issues" >GitHub</a> - Thanks!
</p>
<h2> Description and examples </h2> 
<p>
Click on the buttons to display the fieldnames for the schemas.
</p>
<!-- Tab links -->
<div class="tab">
  <button class="tablinks" onclick="openTab(event, 'by_provider')" id="defaultOpen">By Provider</button>
  <button class="tablinks" onclick="openTab(event, 'by_category')" >By Category</button>

</div>
<!-- Tab content -->
                        <div id="by_category" class="tabcontent">
                        <h3>by_category</h3>
                        <table border="1" class="dataframe table table-striped table-condensed">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Field Name</th>
      <th>description</th>
      <th>example</th>
      <th>used_for</th>
      <th>category</th>
      <th>regex</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>SAMPLE_NAME_GPCF</td>
      <td>Sample ID used within a single ILSe submission for GPCF. Limited to 20 characters.</td>
      <td>0001_hgla_w23T14_sk-3</td>
      <td>ILSe, ODCF</td>
      <td>sample</td>
      <td>[\w-]{1,20}</td>
    </tr>
    <tr>
      <th>1</th>
      <td>PROJECT</td>
      <td>ODCF Project Name as registered in OTP</td>
      <td>OE0538_DO0001</td>
      <td>GUIDE</td>
      <td>sample</td>
      <td>[\w-]{1,32}</td>
    </tr>
    <tr>
      <th>2</th>
      <td>PATIENT_ID</td>
      <td>Individual ID for OTP</td>
      <td>mmus_01234</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>[\w-]{1,32}</td>
    </tr>
    <tr>
      <th>3</th>
      <td>SPECIES</td>
      <td>Scientific name of a species. Note that CAST/Ej is a strain of Mus musculus.</td>
      <td>Mus musculus</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[\w-]{1,64}</td>
    </tr>
    <tr>
      <th>4</th>
      <td>STRAIN</td>
      <td>Identifier for the strain used. Use EFO, if possible</td>
      <td>CAST/Ej, C57BL/6, [CAST/EjxC57BL/6]F1</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>5</th>
      <td>GENOME</td>
      <td>Genome used for QC at GPCF</td>
      <td>mm10</td>
      <td>ILSe</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>6</th>
      <td>INDIVIDUAL</td>
      <td>ID for the individual (MOVI Number, Researcher's working ID)</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>7</th>
      <td>SAMPLE_TYPE</td>
      <td>Sample Type used by OTP, describing samples *across* individual, please follow guidelines</td>
      <td>liver01, liver01-test01</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>8</th>
      <td>TISSUE</td>
      <td>Descriptor of the tissue, if not given it's tried to parse from SAMPLE_TYPE.</td>
      <td>liver, skin, brain</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>9</th>
      <td>GENOTYPE</td>
      <td>Genotype according to EFO. Typically wildtype if not GMO.</td>
      <td>wild type</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>10</th>
      <td>PHENOTYPE</td>
      <td>Description of the phenotype</td>
      <td>healthy, tumor</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>11</th>
      <td>SEX</td>
      <td>Biological sex of the sampled individual.</td>
      <td>female, male, unknown</td>
      <td>ILSe, GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>12</th>
      <td>DATE_OF_BIRTH</td>
      <td>Birthday of individual (format YYYY-MM-DD)</td>
      <td>2020-03-31</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[0-9]{4}-[0-9]{2}-[0-9]{2}</td>
    </tr>
    <tr>
      <th>13</th>
      <td>DATE_OF_DEATH</td>
      <td>Date of death (format YYYY-MM-DD)</td>
      <td>2020-01-04</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[0-9]{4}-[0-9]{2}-[0-9]{2}</td>
    </tr>
    <tr>
      <th>14</th>
      <td>WAY_OF_DEATH</td>
      <td>How was the mouse/animal killed?</td>
      <td>neck_fracture CO2_neck_fracture isofloran_neck_fracture</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>15</th>
      <td>TREATMENT</td>
      <td>Any treatment / pertubation applied to the individual/ cell-line.</td>
      <td>acetone 25w_DMBA/TPA 10w_DMBA/TPA DMBA/TPA_or_TPA TPA UV</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>16</th>
      <td>CELL_INPUT[TOTAL_ALIVE CELLS]</td>
      <td>Number of cell,  if available</td>
      <td>1234</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>[0-9]*</td>
    </tr>
    <tr>
      <th>17</th>
      <td>TISSUE_PREP_METHOD</td>
      <td>Brief description to method of tissue preparation</td>
      <td>liver_ex Single cell MB RNA/DNATissueLyzerII Single cell C. Ernst 1. Enzymes 2. Trypsin 1. Trypsin 2. Enzymes skinbiopsy</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>18</th>
      <td>NA_PREP_METHOD</td>
      <td>Method (probably a kit) used to extract nucleic acid (NA)</td>
      <td>Modified Qiagen DNeasy blood&amp;tissue Qiagen Dneasy blood&amp;tissue Qiagen RNeasy micro</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>19</th>
      <td>NOTES</td>
      <td>Any additional notes, comments for the submission</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>20</th>
      <td>LIB_PREP_KIT</td>
      <td>If using a new Kit, check first with ODCF because they need to register it to their database. Use their descriptor.</td>
      <td>scRNA_10x_v3, WGS_NEBNext_ULTRA_II, Agilent SureSelect Pre-capture, Agilent SureSelect XT Low input Mouse AllExon, Agilent SureSelect Post-capture, NEBNext RNA Low Input - E6420</td>
      <td>GUIDE, Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>21</th>
      <td>AVERAGE_FRAGMENT_SIZE</td>
      <td>Experimentally determined average fragment length (basepairs) if available</td>
      <td>500</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>[0-9]*</td>
    </tr>
    <tr>
      <th>22</th>
      <td>BARCODETYPE</td>
      <td>Description of barcode used</td>
      <td>SureSelect 16 Rxn 10X Chromium v3 NEB 96 oligos SureSelect 96 rxn</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>23</th>
      <td>BARCODE_WELL_I5</td>
      <td>Well position of the barcode</td>
      <td>A7, N701, SI-GA-A6</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>24</th>
      <td>BARCODE_SEQ_I5</td>
      <td>Sequence of the I5 barcode</td>
      <td>TTACCGAC, TATCCTCT</td>
      <td>ILSe, Odomlab</td>
      <td>experiment</td>
      <td>[ATGC]*</td>
    </tr>
    <tr>
      <th>25</th>
      <td>BARCODE_WELL_I7</td>
      <td>Well position of the barcode</td>
      <td>A7, S503</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>26</th>
      <td>BARCODE_SEQ_I7</td>
      <td>Sequence of the I7 barcode (for 10x-singcell this can be a comma-separated list of 4 indeces)</td>
      <td>TTACCGAC, GCGAGTAA</td>
      <td>ILSe, Odomlab</td>
      <td>experiment</td>
      <td>[ATGC, ]*</td>
    </tr>
    <tr>
      <th>27</th>
      <td>DNA_FRAGMENTATION_METHOD</td>
      <td>Method used to fragment DNA</td>
      <td>enzymatic sonicator</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>28</th>
      <td>IMAGING_DATASET_ID</td>
      <td>Imaging dataset for Spatial Transcriptomics</td>
      <td>DOIMG-00001</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>29</th>
      <td>MULTIPLEX_NAME</td>
      <td>Your initials followed by the date of submission format(ABYYYYMMDD)</td>
      <td>MB20190923</td>
      <td>ILSe, Odomlab</td>
      <td>sequencing</td>
      <td>[A-Z]{2}[0-9]{4}[0-9]{2}[0-9]{2}</td>
    </tr>
    <tr>
      <th>30</th>
      <td>ILSE_NO</td>
      <td>ILSE number as provided by GPCF/ILSe</td>
      <td>12345</td>
      <td>GUIDE, Odomlab</td>
      <td>sequencing</td>
      <td>[0-9]{5,}</td>
    </tr>
    <tr>
      <th>31</th>
      <td>SINGLE_CELL_WELL_LABEL</td>
      <td>REQUIRED FOR SMARTSEQ2, combination of column and row</td>
      <td>1A</td>
      <td>GUIDE</td>
      <td>sequencing</td>
      <td>[0-9]{1,2}[A-Z]</td>
    </tr>
    <tr>
      <th>32</th>
      <td>ANTIBODY_TARGET</td>
      <td>Antibody target protein</td>
      <td>HNF4A, CEBPA</td>
      <td>Odomlab</td>
      <td>other</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>33</th>
      <td>ANTIBODY</td>
      <td>Antibody name, manufacturer etc.</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>other</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table>
                        </div><!-- Tab content -->
                        <div id="by_provider" class="tabcontent">
                        <h3>by_provider</h3>
                        <table border="1" class="dataframe table table-striped table-condensed">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Field Name</th>
      <th>description</th>
      <th>example</th>
      <th>used_for</th>
      <th>category</th>
      <th>regex</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>SAMPLE_NAME_GPCF</td>
      <td>Sample ID used within a single ILSe submission for GPCF. Limited to 20 characters.</td>
      <td>0001_hgla_w23T14_sk-3</td>
      <td>ILSe, ODCF</td>
      <td>sample</td>
      <td>[\w-]{1,20}</td>
    </tr>
    <tr>
      <th>1</th>
      <td>GENOME</td>
      <td>Genome used for QC at GPCF</td>
      <td>mm10</td>
      <td>ILSe</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>2</th>
      <td>SEX</td>
      <td>Biological sex of the sampled individual.</td>
      <td>female, male, unknown</td>
      <td>ILSe, GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>3</th>
      <td>BARCODE_SEQ_I5</td>
      <td>Sequence of the I5 barcode</td>
      <td>TTACCGAC, TATCCTCT</td>
      <td>ILSe, Odomlab</td>
      <td>experiment</td>
      <td>[ATGC]*</td>
    </tr>
    <tr>
      <th>4</th>
      <td>BARCODE_SEQ_I7</td>
      <td>Sequence of the I7 barcode (for 10x-singcell this can be a comma-separated list of 4 indeces)</td>
      <td>TTACCGAC, GCGAGTAA</td>
      <td>ILSe, Odomlab</td>
      <td>experiment</td>
      <td>[ATGC, ]*</td>
    </tr>
    <tr>
      <th>5</th>
      <td>MULTIPLEX_NAME</td>
      <td>Your initials followed by the date of submission format(ABYYYYMMDD)</td>
      <td>MB20190923</td>
      <td>ILSe, Odomlab</td>
      <td>sequencing</td>
      <td>[A-Z]{2}[0-9]{4}[0-9]{2}[0-9]{2}</td>
    </tr>
    <tr>
      <th>6</th>
      <td>PROJECT</td>
      <td>ODCF Project Name as registered in OTP</td>
      <td>OE0538_DO0001</td>
      <td>GUIDE</td>
      <td>sample</td>
      <td>[\w-]{1,32}</td>
    </tr>
    <tr>
      <th>7</th>
      <td>PATIENT_ID</td>
      <td>Individual ID for OTP</td>
      <td>mmus_01234</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>[\w-]{1,32}</td>
    </tr>
    <tr>
      <th>8</th>
      <td>SAMPLE_TYPE</td>
      <td>Sample Type used by OTP, describing samples *across* individual, please follow guidelines</td>
      <td>liver01, liver01-test01</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>9</th>
      <td>PHENOTYPE</td>
      <td>Description of the phenotype</td>
      <td>healthy, tumor</td>
      <td>GUIDE, Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>10</th>
      <td>LIB_PREP_KIT</td>
      <td>If using a new Kit, check first with ODCF because they need to register it to their database. Use their descriptor.</td>
      <td>scRNA_10x_v3, WGS_NEBNext_ULTRA_II, Agilent SureSelect Pre-capture, Agilent SureSelect XT Low input Mouse AllExon, Agilent SureSelect Post-capture, NEBNext RNA Low Input - E6420</td>
      <td>GUIDE, Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>11</th>
      <td>ILSE_NO</td>
      <td>ILSE number as provided by GPCF/ILSe</td>
      <td>12345</td>
      <td>GUIDE, Odomlab</td>
      <td>sequencing</td>
      <td>[0-9]{5,}</td>
    </tr>
    <tr>
      <th>12</th>
      <td>SINGLE_CELL_WELL_LABEL</td>
      <td>REQUIRED FOR SMARTSEQ2, combination of column and row</td>
      <td>1A</td>
      <td>GUIDE</td>
      <td>sequencing</td>
      <td>[0-9]{1,2}[A-Z]</td>
    </tr>
    <tr>
      <th>13</th>
      <td>DATE_OF_BIRTH</td>
      <td>Birthday of individual (format YYYY-MM-DD)</td>
      <td>2020-03-31</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[0-9]{4}-[0-9]{2}-[0-9]{2}</td>
    </tr>
    <tr>
      <th>14</th>
      <td>CELL_INPUT[TOTAL_ALIVE CELLS]</td>
      <td>Number of cell,  if available</td>
      <td>1234</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>[0-9]*</td>
    </tr>
    <tr>
      <th>15</th>
      <td>IMAGING_DATASET_ID</td>
      <td>Imaging dataset for Spatial Transcriptomics</td>
      <td>DOIMG-00001</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>16</th>
      <td>NOTES</td>
      <td>Any additional notes, comments for the submission</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>17</th>
      <td>AVERAGE_FRAGMENT_SIZE</td>
      <td>Experimentally determined average fragment length (basepairs) if available</td>
      <td>500</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>[0-9]*</td>
    </tr>
    <tr>
      <th>18</th>
      <td>DNA_FRAGMENTATION_METHOD</td>
      <td>Method used to fragment DNA</td>
      <td>enzymatic sonicator</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>19</th>
      <td>ANTIBODY_TARGET</td>
      <td>Antibody target protein</td>
      <td>HNF4A, CEBPA</td>
      <td>Odomlab</td>
      <td>other</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>20</th>
      <td>BARCODE_WELL_I7</td>
      <td>Well position of the barcode</td>
      <td>A7, S503</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>21</th>
      <td>TREATMENT</td>
      <td>Any treatment / pertubation applied to the individual/ cell-line.</td>
      <td>acetone 25w_DMBA/TPA 10w_DMBA/TPA DMBA/TPA_or_TPA TPA UV</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>22</th>
      <td>TISSUE</td>
      <td>Descriptor of the tissue, if not given it's tried to parse from SAMPLE_TYPE.</td>
      <td>liver, skin, brain</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>23</th>
      <td>STRAIN</td>
      <td>Identifier for the strain used. Use EFO, if possible</td>
      <td>CAST/Ej, C57BL/6, [CAST/EjxC57BL/6]F1</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>24</th>
      <td>WAY_OF_DEATH</td>
      <td>How was the mouse/animal killed?</td>
      <td>neck_fracture CO2_neck_fracture isofloran_neck_fracture</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>25</th>
      <td>INDIVIDUAL</td>
      <td>ID for the individual (MOVI Number, Researcher's working ID)</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>26</th>
      <td>BARCODETYPE</td>
      <td>Description of barcode used</td>
      <td>SureSelect 16 Rxn 10X Chromium v3 NEB 96 oligos SureSelect 96 rxn</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>27</th>
      <td>NA_PREP_METHOD</td>
      <td>Method (probably a kit) used to extract nucleic acid (NA)</td>
      <td>Modified Qiagen DNeasy blood&amp;tissue Qiagen Dneasy blood&amp;tissue Qiagen RNeasy micro</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>28</th>
      <td>DATE_OF_DEATH</td>
      <td>Date of death (format YYYY-MM-DD)</td>
      <td>2020-01-04</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[0-9]{4}-[0-9]{2}-[0-9]{2}</td>
    </tr>
    <tr>
      <th>29</th>
      <td>GENOTYPE</td>
      <td>Genotype according to EFO. Typically wildtype if not GMO.</td>
      <td>wild type</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>30</th>
      <td>TISSUE_PREP_METHOD</td>
      <td>Brief description to method of tissue preparation</td>
      <td>liver_ex Single cell MB RNA/DNATissueLyzerII Single cell C. Ernst 1. Enzymes 2. Trypsin 1. Trypsin 2. Enzymes skinbiopsy</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>31</th>
      <td>BARCODE_WELL_I5</td>
      <td>Well position of the barcode</td>
      <td>A7, N701, SI-GA-A6</td>
      <td>Odomlab</td>
      <td>experiment</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>32</th>
      <td>ANTIBODY</td>
      <td>Antibody name, manufacturer etc.</td>
      <td>None</td>
      <td>Odomlab</td>
      <td>other</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>33</th>
      <td>SPECIES</td>
      <td>Scientific name of a species. Note that CAST/Ej is a strain of Mus musculus.</td>
      <td>Mus musculus</td>
      <td>Odomlab</td>
      <td>sample</td>
      <td>[\w-]{1,64}</td>
    </tr>
  </tbody>
</table>
                        </div>
<script>
function openTab(evt, TabName) {
  var i, tabcontent, tablinks;
  tabcontent = document.getElementsByClassName("tabcontent");
  for (i = 0; i < tabcontent.length; i++) {
    tabcontent[i].style.display = "none";
  }
  tablinks = document.getElementsByClassName("tablinks");
  for (i = 0; i < tablinks.length; i++) {
    tablinks[i].className = tablinks[i].className.replace(" active", "");
  }
  document.getElementById(TabName).style.display = "block";
  evt.currentTarget.className += " active";
}
// Get the element with id="defaultOpen" and click on it
document.getElementById("defaultOpen").click();
</script>
</body>
</html>
